config/src/config/test_data/validator.yaml

```diff
@@ -38,7 +38,6 @@ full_node_networks:
           private: "vfn"
 
 validator_network:
-    discovery_method: "onchain"
     listen_address: "/ip4/0.0.0.0/tcp/6180"
```

Comment:
Don't we need to prepend `listen_address` with `-`? Looks like it was removed with the line deletion, but I'm not sure if the YAML parser will be happy now 😄

Reply:
This tripped me up too. GitHub shows a `-` to indicate that the line was removed in my commit.

I double checked to make sure that I preserved the `-` if it was in the original file